### PR TITLE
Update the changes utils to be a requireable module

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8068/

File: jujugui/static/gui/src/app/init/changes-utils.js [hunk 1]
```diff
@@ -1,62 +1,40 @@
...
 'use strict';
 
-YUI.add('changes-utils', function(Y) {
-
-  var ChangesUtils = {};
-
-  var removeBrackets = /^\(?(.{0,}?)\)?$/;
+class ChangesUtils {
+  constructor() {
+    this.removeBrackets = /^\(?(.{0,}?)\)?$/;
```
Comment (hatched) [inline jujugui/static/gui/src/app/init/changes-utils.js]: I think I'd have this as a constant defined outside the class....unless for some reason you need this accessible on the instance itself?

File: jujugui/static/gui/src/app/init/changes-utils.js [hunk 2]
```diff
@@ -1,62 +1,40 @@
...
-
+/* Copyright (C) 2017 Canonical Ltd. */
 'use strict';
 
-YUI.add('changes-utils', function(Y) {
-
-  var ChangesUtils = {};
-
-  var removeBrackets = /^\(?(.{0,}?)\)?$/;
+class ChangesUtils {
```
Comment (hatched) [inline jujugui/static/gui/src/app/init/changes-utils.js]: Why did you create a class for this instead of having it as a collection of methods as before?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8069/

**Pull request merged**